packages/webapp/src/components/Crop/Detail.jsx

```diff
@@ -38,6 +39,12 @@ function PureCropDetail({
   const GENETICALLY_ENGINEERED = 'genetically_engineered';
   const HS_CODE_ID = 'hs_code_id';
   const isOrganic = isEditing ? watch(ORGANIC) : variety.organic;
+  const scientificNameLabel = variety.crop_genus
+    ? variety.crop_genus
+    : '' + ' ' + variety.crop_specie
+    ? variety.crop_specie
+    : '';
```

Comment:
I don't think this works as you intended. When there is `variety.crop_genus`, `variety.crop_specie` is not shown.
It might be better to create a function.